themes/digital.gov/src/scss/new/_sitewide-alert.scss

```
@@ -0,0 +1,4 @@
+// set margin-top to 0 to leave no space between banner and alert
+.dg-site-alert .usa-alert {
+  margin-top: 0rem;
```

You can leave out units on values of zero.

```suggestion
  margin-top: 0;
```